src/Handlers/ShowHandler.ts

```diff
@@ -11,6 +11,7 @@ import { HandlerTypes, HookFunctionTypes } from "../Enums";
 import ApiError from "../Exceptions/ApiError";
 import { QueryService } from "../Services";
 import { Knex } from "knex";
+import { StatusCodes } from "http-status-codes";
```

Comment:
I remember that I already removed the `http-status-codes` package from dependencies. You can find the `StatusCodes` enum in the `Enum.ts` file. Can you use it?